gulpfile.js

```diff
@@ -170,11 +170,17 @@ function delRev(dest, manifestName) {
 
 gulp.task('serve', function(cb) {
   let called = false;
+  let execParams = path.normalize('node_modules/.bin/babel-node');
+  // When in development we can spawn a node debugger
+  // https://nodejs.org/en/docs/inspector/
+  if (__DEV__) {
+    execParams = execParams + ' --inspect';
```

Comment:
Why inspect and not inspect-brk?

Reply:
Nice question. 

This is because the `--inspect-brk` flag will block all further execution until a debugger is attached and you manually click the proceed button (or press <kbd>F8</kbd>, etc.) to move past the first point of break.

This will be very annoying when we are not interested in debugging with breakpoints. For example, editing challenges etc.

Reply:
In most cases we would want to debug, the server side code once its up and running and is either receiving or ready to receive requests.

For the corner and rare cases of pre-ready stage, we can locally edit this file OR directly run the server with the required debug flags.

My intent is to have a quick one click open the dev tools and start to debug solution.